boards/arm/olimex_stm32_e407/doc/olimex_stm32_e407.rst

```diff
@@ -0,0 +1,383 @@
+
+PF
+
++-------+--------------+-------+--------------+
+| PIN # | Signal Name  | Pin # | Signal Name  |
++=======+==============+=======+==============+
+| 1     | +3.3V        | 11    | PF8/A3       |
++-------+--------------+-------+--------------+
+| 2     | GND          | 12    | PF9/A4       |
++-------+--------------+-------+--------------+
+| 3     | PF0          | 13    | PF10/A5      |
++-------+--------------+-------+--------------+
+| 4     | PF1          | 14    | PF11/A6      |
++-------+--------------+-------+--------------+
+| 5     | PF2          | 15    | PF12         |
++-------+--------------+-------+--------------+
+| 6     | PF3          | 16    | PF13         |
++-------+--------------+-------+--------------+
+| 7     | PF4          | 17    | PF14         |
++-------+--------------+-------+--------------+
+| 8     | PF5          | 18    | PF15         |
++-------+--------------+-------+--------------+
+| 9     | PF6/A1       | 19    | +5V          |
++-------+--------------+-------+--------------+
+| 10    | PF7/A2       | 20    | GND          |
++-------+--------------+-------+--------------+
+
+PG
+
++-------+--------------+-------+--------------+
+| PIN # | Signal Name  | Pin # | Signal Name  |
++=======+==============+=======+==============+
+| 1     | +3.3V        | 11    | PG8/D7       |
++-------+--------------+-------+--------------+
+| 2     | GND          | 12    | PG9          |
++-------+--------------+-------+--------------+
+| 3     | PG0          | 13    | PG10/UEXT_CS |
++-------+--------------+-------+--------------+
+| 4     | PG1          | 14    | PG11/TX_EN   |
++-------+--------------+-------+--------------+
+| 5     | PG2          | 15    | PG12/D8      |
++-------+--------------+-------+--------------+
+| 6     | PG3          | 16    | PG13/TXD0    |
++-------+--------------+-------+--------------+
+| 7     | PG4          | 17    | PG14/TXD1    |
++-------+--------------+-------+--------------+
+| 8     | PG5          | 18    | PG15/D9      |
++-------+--------------+-------+--------------+
+| 9     | PG6          | 19    | +5V          |
++-------+--------------+-------+--------------+
+| 10    | PG7/D6       | 20    | GND          |
++-------+--------------+-------+--------------+
+
+System Clock
+============
+
+OLIMEX-STM32-E407 has two external oscillators. The frequency of
+the slow clock is 32.768 kHz. The frequency of the main clock
+is 12 MHz. The processor can setup HSE to drive the master clock,
+which can be set as high as 168 MHz.
+
+Programming and Debugging
+=========================
+The OLIMEX-STM32-E407 board does not include an embedded debug tool
+interface. You will need to use ST tools or an external JTAG probe.
+In the following examples a ST-Link V2 USB dongle is used.
+
+Flashing an application to STM32F4DISCOVERY
+-------------------------------------------
+
+The sample application :ref:`hello_world` is being used in this tutorial:
+
+.. code-block:: console
```

Comment:
delete this code block (and change the colon after tutorial above to a period).  The :ref:`hello_world` provides information about the hello_world app and where to find it.